### ISSUE TITLE
Loading a file without video causes MPC-HC to crash

### ISSUE THREAD

**Issue by CrazyPyroEagle** [SubtitleEdit/subtitleedit#3193]
With MPC-HC selected as the video player, loading an audio file or a container file without a video stream causes "MPC-HC has stopped working" to popup instead of the expected "missing video" warning.

Slightly unrelated question, but why is subtitle editing not possible without a video stream?

**Comment by niksedk** [comment]
Hm, it seems to work here...
What happens if you try another player?

**Comment by OmrSi** [comment]
Well, I tested here Using FFmpeg...


When you add a valid WAV file (I added a WAV from the files in Waveform folder and tested some mp3 files which I just changed their extension from mp3 to WAV)

If MPV or VLC or DirectShow is chosen:
The file opens fine with no video with no problem.
![screenshot 743](https://user-images.githubusercontent.com/20923700/48972862-0953f000-f03c-11e8-87c7-3f6334008137.png)

BUT, if you add another file while another one is already loaded, this happens in the three of them:
![screenshot 748](https://user-images.githubusercontent.com/20923700/48973086-2f7b8f00-f040-11e8-8f6f-cefe746eb4bb.png)

P.S: if you use DirectShow and load a file from the Waveform folder, when you play it, it just goes to the end of the file.

else If MPC is chosen as the video player:
![screenshot 744](https://user-images.githubusercontent.com/20923700/48972878-3d2f1580-f03c-11e8-93f2-f2918c152369.png)
Then the warning goes away and the file is loaded but everytime you press Play it doesn't play:
![screenshot 745](https://user-images.githubusercontent.com/20923700/48972890-8b441900-f03c-11e8-82ab-eb17fcaea803.png)



On an Unrelated note: Why is the cursor movement on the waveform when DirectShow is chosen smoother than MPV?
